demos/scenes/scenes.go

```diff
@@ -30,7 +30,7 @@ func (game *IconScene) Setup(w *ecs.World) {
 	w.AddSystem(&ScaleSystem{})
 	w.AddSystem(&SceneSwitcherSystem{NextScene: "RockScene", WaitTime: time.Second * 3})
 
-	guy := ecs.NewEntity({"RenderSystem", "ScaleSystem")
+	guy := ecs.NewEntity("RenderSystem", "ScaleSystem")
```

Comment:
This is indeed supposed to change; although it's already changed in the `master` branch.